### ISSUE TITLE
Expo not marked as optional peer

### ISSUE THREAD

**Issue by abhishiv** [tinyplex/tinybase#159]
### Describe the bug

Hello, @tinyplex

Thank you very much for a great package. 

Just FYI tinybase seems to pull in around 700 dependencies including stuff like expo

```
➜  pkg git:(refactor) ✗ npm uninstall tinybase -w xxx              
                                                                                     
removed 714 packages, and audited 523 packages in 2s
```

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by jamesgpearce** [comment]
Everything* is an optional peer, or developer, dependency.

* BUT I did notice that for some reason expo (a peer dependency) is not marked optional... so let me fix that!

**Comment by jamesgpearce** [comment]
Yep, that fixed it! Please update to v5.0.4, delete your package-lock.json, and node_modules directory. Install again and you should have no dependencies brought in by TinyBase!

**Comment by jamesgpearce** [comment]
A bit embarrassing TBH. Thanks for finding! 🥰
